example_governance_doc.md

```diff
@@ -43,17 +43,56 @@ These stakeholders comprise the Schema Working Group, or SWG.
 
 ## Getting Help
 
-If you're not familiar with GraphQL, please start by looking at [Apollo's Odyssey tutorials](https://apollographql.com/tutorials/) to get acquainted with the technology.
-
 To request help with the ACME graph, please reach out in #graphql in Slack. 
 
+The following is a proposed learning path for contributors not familiar with Apollo GraphQL.
+
+### Learning Paths
+
+Apollo offers two certification courses through their Odyssey platform: [Associate](https://www.apollographql.com/tutorials/certifications/apollo-graph-associate) and [Professional](https://www.apollographql.com/tutorials/certifications/apollo-graph-professional). Contributors are recommended to complete at least the Associate certification in order to understand the basics of GraphQL and Federation.
```

Comment:
We have the new API orchestration associate path that should be recommended first: https://www.apollographql.com/tutorials/certifications/apollo-orchestration-associate

Reply:
@R4ph-t is this a replacement for the Associate certification? to clarify - the goal here is to get a foundational understanding of graphQL in general and federation in particular. from my understanding the orchestration certification focusses more on REST connectors. if these are different i'd want to list them all out and describe the use case for each certification